### DISCUSSION TITLE
One of the best alternatives of NitroShare

### DISCUSSION THREAD

**Discussion by DeltaGRaptor** [slowscript/warpinator-windows#5] [Ideas]
I was searching for the NitroShare replacement, that works on Windows, Android, and across different platforms. I then stumbled on Warpinator. As it is an official app for the Linux distros, I kept searching for the alternative and found this app. It is truly a better replacement for NitroShare, at least for Windows users, because NitroShare does not yet support duplex transfers. Thanks for developing this app. My request is that you add "Share with" feature in the context menu and "Auto start after Login", that would be really useful. Thanks.

**Comment by slowscript** [comment]
Thank you for your feedback. I've now added the ability to open and send files via the "Send To" context menu in Windows Explorer (if this is what you meant). The integration itself will be done by the installer or can be easily done manually later. You can already do autostart after login yourself by following [these instructions](https://support.microsoft.com/en-us/windows/add-an-app-to-run-automatically-at-startup-in-windows-10-150da165-dcd9-7230-517b-cf3c295d89dd). Same as with the Send To integration (if you have version 0.2), follow the same steps, just paste the link in `shell:sendto` instead of `shell:startup`.
